src/displayapp/screens/Twos.cpp

```diff
@@ -151,28 +144,30 @@ bool Twos::tryMove(TwosTile grid[][4], int newRow, int newCol, int oldRow, int o
 
 bool Twos::OnTouchEvent(Pinetime::Applications::TouchEvents event) {
   bool validMove = false;
-  for (int row = 0; row < 4; row++) {
-    for (int col = 0; col < 4; col++) {
-      grid[row][col].merged = false; // reinitialize merge state
-    }
+  for (unsigned int i = 0; i < nCells; i++) {
+    const unsigned int row = i / nCols;
+    const unsigned int col = i % nCols;
```

Comment:
why do a single loop and a possibly costly modulo operation instead of a nested loop like done further down?

Reply:
It saves a few bytes of memory even when using a 5x5 grid.